.github/workflows/release.yml

```diff
@@ -77,10 +83,11 @@ jobs:
             --entrypoint /bin/bash \
             ${{ env.DOCKER_ORG }}/pulumi:${{ env.PULUMI_VERSION }} \
             -c "/src/pulumi-test-containers -test.parallel=1 -test.timeout=1h -test.v -test.run TestPulumiDockerImage"
-      - name: Push image
-        run: |
-          docker push ${{ env.DOCKER_ORG }}/pulumi:${{ env.PULUMI_VERSION }}
-          docker push ${{ env.DOCKER_ORG }}/pulumi:latest
+      - name: Push ${{ env.PULUMI_VERSION }}
+        run: docker push ${{ env.DOCKER_ORG }}/pulumi:${{ env.PULUMI_VERSION }}
+      - name: Push latest
+        if: ${{ github.event.inputs.tag_latest || github.event_name == 'repository_dispatch' }}
```

Comment:
`tag_latest` now defaults to `true`.  Marking resolved.

Reply:
A clarifying question here: what are the `event.inputs` in case the even isn't a release dispatch?

do we want `event.inputs.tag_latest` to mean "potentially an older version"? Or  would the repo dispatch be potentially an older version?

Reply:
`repository_dispatch` only has the Pulumi version (in the `ref` field).  pulumictl can't currently support multiple fields when it fires the event, but this is ok because an automated run of this workflow (i.e. `repository_dispatch`) will always want to tag latest because it's triggered by a release of the Pulumi CLI.

`workflow_dispatch` (i.e. manual run) is what has `tag_latest` as an option.

Reply:
Ah, I see.

But what is the use case for `workflow_ dispatch` to _not_ build `latest`?

Reply:
For example, we need to rebuild an earlier version because we miss a release for some reason.  It's unlikely we'll need to, but I wanted to make sure we provided the levers to do so if necessary.

Reply:
I suppose I'm not familiar with the use case of `latest` out in the wild - I'd expect `latest` to be unstable, and only be concerned with making sure we have all semver tagged images to be distinct - which is covered on line 88 and the fact we can manually supply the semver via workflow dispatch?